ui/package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "test": "vitest spec",
-    "test:e2e": "EXTENSION=keplr synpress run --configFile=test/e2e/synpress.config.cjs",
+    "test:e2e": "SKIP_EXTENSION_SETUP=true EXTENSION=keplr synpress run --configFile=test/e2e/synpress.config.cjs",
```

Comment:
how does the account with the `spike siege world rather ...` mnemonic get set up?

Reply:
Are you trying to set up this account locally, Dan?

Reply:
no; I'm trying to understand how `test:e2e` works when skipping extension set-up.

Reply:
You don't need to set up an account locally using the `spike siege word rather...` mnemonic. However, if you want to run the test cases with the `spike siege word rather...` mnemonic, you can replace `secretWords` [here](https://github.com/Agoric/dapp-offer-up/blob/main/ui/test/e2e/specs/test.spec.js#L7C5-L8C158).

Reply:
Normally, when we don't pass the `SKIP_EXTENSION_SETUP` flag, it executes the `setupWallet` method inside the `before` hook present in this [file](https://github.com/agoric-labs/synpress/blob/dev/support/index.js#L27-L37C15) of `@agoric/synpress`. 

When we pass `SKIP_EXTENSION_SETUP` and set it to `true` in `test:e2e`, the `setupWallet` function is not executed inside the `before` hook.

Reply:
The implementation for `setupWallet` is defined [over here](https://github.com/agoric-labs/synpress/blob/dev/support/commands.js#L413-L434)

It tends to use a hard-coded mnemonic `orbit bench unit...` if we don't pass `secretWords` to the `setupWallet`.

I am going to eliminate this though. Instead of using a hard-coded value, we'll use the one passed via the environment variable. Right now, we accept the environment variable(`SECRET_WORDS`) for mnemonic but don't use its value. [HERE](https://github.com/agoric-labs/synpress/issues/40).